Engine/src/Engine/Physics/AABB.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "glm/glm.hpp"	
+#include "Ray.h"
+
+namespace Engine
+{
+	/// <summary>
+	/// Wrapper for corner indicies
+	/// </summary>
+	enum Corner
+	{
+		LTF = 0, // Left-Top-Front

```

Comment:
Why is LTF the only enum with 0 as the instantiated value? This is likely just my lower understanding of C++.

Reply:
You can assign integer/hex values to enums and reference them as such. In this instance LTF is the first corner/cube in the order of traversal so it is index 0, RBB is the final corner/cube and as such is index 7. This is used in a lot of the AABB referencing code.

Reply:
Ok awesome, thanks for the explanation!